### PR TITLE
cmd/explain: make 'recursive' local var (not global)

### PR DESCRIPTION
**Pull request by alexandercampbell**
**What this PR does / why we need it**:
Use a parameter instead of a global variable.
Spotted this while I was looking for our recursive directory walker.

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: no-issue

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: pkg/kubectl/explain.go [hunk 1]
```diff
@@ -38,8 +37,7 @@ func SplitAndParseResourceRequest(inResource string, mapper meta.RESTMapper) (st
 }
 
 // PrintModelDescription prints the description of a specific model or dot path
```
Comment (ericchiang) [inline pkg/kubectl/explain.go]: Can you add a comment to the doc string explaining what the recursive argument does?

**Review by ericchiang — APPROVED** [review]
thanks! lgtm other than one nit

File: pkg/kubectl/explain.go (hunk 1 above)
Comment (alexandercampbell) [inline pkg/kubectl/explain.go]: Updated. Thanks!

**Pull request merged**